packages/next/src/build/swc/index.ts

```diff
@@ -507,7 +507,7 @@ export interface Issue {
       ident: string
       content?: string
     }
-    range?: {
+    range: null | {
```

Comment:
Is it really `null`? I would guess `serde` would serialize `Option<T>` as skipping that property.